sfcasts/bundle-config.md

```diff
@@ -1,4 +1,35 @@
-Now you can see that we use `array adapter` for our `app cache`. Let's remove the `dd($cache)` and see the `array cache adapter` in action. Refresh the page again. Actually, every time you refresh the page, we execute the `HTTP request`. So the cache is only live during the request. When I start a new request, cache invalidates and we see that `HTTP` requests.
+So far, we've learned how to use the HTTP Client and cache services, and we've injected that into the homepage. *But*, we're not responsible for *creating* their objects. We already know that bundles give us services, and when we *autowire* a service, our bundle provides all of the details we need to instantiate it. But if something *else* is responsible for instantiating those objects, how can we control it? The answer is *bundle configuration*.
+
+Open the `/config/packages` directory. All of these `.yaml` config files are *automatically* loaded into our Symfony application, and their job is to configure services that each bundle give us. In our `homepage()`, after `Response`, let's `dd($cache)` so we can see the class name of the object we're getting. For example, for the cache service, `FrameworkBundle` tells the service container:
+
+`Hey! When I ask for the CacheInterfaceService, I
+want you to instantiate this "TraceableAdapter"
+object with a specific set of arguments.`
+
+So it looks like our cache is just this `TraceableAdapter`, but if we look *closer*, we can see that it's *actually* a wrapper around a file system, and the cache is stored *inside* the file system. That's cool, but what if we want to store the cache in memory instead? Or somewhere else in the file system? This is where bundle configuration *shines*. Open `framework.yaml` and find this `framework` key. This means we're passing configuration to the `FrameworkBundle`, and it'll *use* that config to change how it instantiates its services. By the way, the file name here isn't important. We *could* call this `pizza.yaml` and it would work just the same.
```

Comment:
```suggestion
So it looks like our cache service is just this `TraceableAdapter`, but if we look *closer*, we can see that it's *actually* a wrapper around a `FilesystemAdapter`, and the cache is stored *inside* the file system. That's cool, but what if we want to store the cache in memory instead? Or somewhere else in the file system? This is where bundle configuration *shines*. Open `framework.yaml` and find this `framework` root key. This means we're passing configuration to the `FrameworkBundle`, and it'll *use* that config to change how it instantiates its services. By the way, the file name here isn't important. We *could* call this `pizza.yaml` and it would work just the same.
```